public/process_news_stories.html

```diff
@@ -47,7 +47,7 @@ <h1>Notion to Geo Publisher</h1>
         placeholder="Enter wallet address"
       />
     </div>
-    <button id="processBtn">Publish</button>
+    <button id="processBtn">Remove duplciates</button>
```

Comment:
Wrong file edited, I assume

```suggestion
    <button id="processBtn">Publish news stories</button>
```